maestro-client/src/main/java/maestro/ViewHierarchy.kt

```diff
@@ -100,6 +101,10 @@ value class ViewHierarchy(val root: TreeNode) {
 }
 
 fun TreeNode.filterOutOfBounds(width: Int, height: Int): TreeNode? {
+    if (attributes.containsKey("class") && attributes["class"] == TOAST_CLASS_NAME) {
+        attributes.remove("class")
+        return this
+    }
```

Comment:
I hate doing this, Android-based logic in ViewHierarchy class. The problem is that we don't have bounds of toast. And since we cannot get bounds, it gets filters here. Is there any better way that we can use to make exceptions for Toast and still make this class platform agnostic?

Reply:
Good idea done 💯